### DISCUSSION TITLE
Bootstrap

### DISCUSSION THREAD

**Discussion by mishelshaji** [mishelshaji/Learn-DotNet#18] [Progress Tracker]
- What is Bootstrap?
- Downloading bootstrap.
- Adding bootstrap using CDN.
- Bootstrap components
    - Alert
    - Button
    - Card
    - Table
    - Navbar
    - Progress bars
    - Spinners
    - Badge
    - Form and form controls
    - Button group
    - List group
- Utilities
    - Margin
    - Padding
    - Shadow
    - Border
    - Spacing
    - Display
- Bootstrap grid system
- Building forms
- Styling elements

**Comment by Jiby-Joseph-123** [comment]
Completed.

**Comment by sreelakshmiChandran** [comment]
Completed

**Comment by riya45810** [comment]
Completed.

**Comment by Arjunprasad-22** [comment]
Completed

**Comment by ManikuttyBabu** [comment]
completed

**Comment by Shamla-P** [comment]
Completed

**Comment by AshinDamodaran** [comment]
completed

**Comment by SonuSanilK** [comment]
Completed

**Comment by Nandana-04** [comment]
Completed

**Comment by jeweljose22** [comment]
Completed

**Comment by kavitha-BS** [comment]
Completed

**Comment by AnitaMaryThomas** [comment]
Completed

**Comment by SivapriyaVB** [comment]
Completed
